src/components/PersonLinks/PersonLinks.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { Person } from '../../types';
+import { Link, useParams } from 'react-router-dom';
+import classNames from 'classnames';
+
+type Props = {
+  person: Person;
+  people: Person[];
+};
+
+export const PersonLinks: React.FC<Props> = ({ person, people }) => {
+  const { slug: id } = useParams();
```

Comment:
The `useParams` hook returns an object with values that are always strings or undefined. Ensure that `id` is correctly handled as a string, and consider adding type annotations to avoid potential runtime errors.